### ISSUE TITLE
Is information about the Advisory Team under "Community" accurate?

### ISSUE THREAD

**Issue by pdurbin** [IQSS/dataverse.org#63]
I've mentioned this internally here and there but it's a new year and I still don't love what newcomers to the project experience when they click "Community" on dataverse.org, especially with regard to "Advisory Team".

To be clear, I think it's fantastic that the word "Community" is so prominent on the project home page. I think it's fine that we highlight our community meetings, mailing list, and community calls, but I don't like that the first thing under "Community" is "Advisory Team". Here's how it looks:

<img width="1159" alt="screen shot 2018-01-11 at 7 28 55 am" src="https://user-images.githubusercontent.com/21006/34826063-1a8b2264-f6a3-11e7-99df-7db99c0b0d78.png">

The person clicking "Community" is probably thinking, "This Advisory Team must be pretty important since it's listed first." They click and they read the following:

>  Advisory Team
> 
> As an open source project, the Dataverse team is dedicated to working with all interested stakeholders of the research community to help in defining future development efforts to continue expanding and enhancing the software application. In addition to our Users forum, **there is an Advisory Team which meets quarterly**, and an annual Community Meeting.
> 
> The Dataverse Advisory team provides advice on technical directions and good practices to the Dataverse team at IQSS, Harvard University. Members of the Advisory team are external to IQSS, and should manage a Dataverse installation and/or be active users and participants in the Dataverse community.
> 
> Charges: The Dataverse Advisory team advises the Dataverse team at IQSS, on all significant general areas involving the Dataverse software application.
> 
> Member of the Advisory Team, **2015-2016**:
> 
> Jonathan Crabtree, ODUM Institute at UNC
> Peter Doorn, Data Archiving and Networked Services at the Netherlands
> John Daniels, Social Science Data Services at UC Davis
> August Muench, American Astronomical Society
> Yin Shenqin,  Social Science Data Center, Fudan University 
> Alan Darnell, Scholars Portal, University of Toronto
> 
> Questions? Please Contact Us and subscribe to our Users forum to find out more about how you can get more involved in the Dataverse Community.

I've highlighted a couple items above. First, if this advisory team is meeting quarterly, what are the outputs of these quarterly meetings? As a member of the Dataverse community am I permitted to know roughly what the advisory team has been talking about? Are there any meeting notes? Or are these completely closed-door meetings?

Second, why do we only see the advisory team for the 2015-2016 academic year? It's 2018 and I'm left wondering if the same people are on the advisory team. Are these positions for life? If appointments to the advisory team are for a year or two, who's on the advisory team now? What's the process for getting on the advisory team?

In short, I'm left with many questions and feeling slightly unsettled when I click "Community" and try to understand the role of the Advisory Team.

When people click "Community" I want people to learn that Dataverse has a passionate, diverse, international, welcoming community that is open to ideas and collaboration. We could say this. We could show the map of 30 installations around the world. We could highlight successful collaborations and pull requests. We can tell a better story about community.

**Comment by pdurbin** [comment]
Ok, removed. Again, that "Advisory Team" menu item brings up more questions than it answers. People interested in this topic should check out the new http://dataversecommunity.global website. Closing.
